draft-ietf-dnsop-rfc7958bis.mkd

```diff
@@ -414,6 +414,11 @@ retrieving the trust anchors trusts the CA that IANA uses for the
 "data.iana.org" web server, HTTPS SHOULD be used instead of HTTP in
 order to have assurance of data origin.
 
+A KeyDigest element can contain both a Digest and a PublicKey element.
+If the Digest element would not be a proper DS record for a DNSKEY record represented by the PublicKey element,
```

Comment:
Wrapping?

Reply:
Wrapping doesn't matter on the Markdown file: it comes out fine when running it through kramdown-rfc.